src/backend/mi2/mi2.ts

```diff
@@ -589,9 +589,9 @@ export class MI2 extends EventEmitter implements IBackend {
 			let location = "";
 			if (breakpoint.countCondition) {
 				if (breakpoint.countCondition[0] == ">")
-					location += "-i " + numRegex.exec(breakpoint.countCondition.substring(1))[0] + " ";
+					location += "-i " + numRegex.exec(breakpoint.countCondition.substring(1))?.[0] ?? "" + " ";
```

Comment:
we should move the `numRegex.exec(...)` into a separate variable, then check that it matches, then either it must match and otherwise print an error and do nothing, or fall back to some value like `0`

Since we are using `-i`, an argument after it is required and we can't just leave it blank like that. (haven't checked GDB documentation to actually verify this statement though, it just looks like it would work that way)

Reply:
done